src/main.rs

```diff
@@ -29,19 +29,19 @@ fn main() {
                 .long("input-format")
                 .required(false)
                 .value_parser(["json", "yaml"])
-                .default_value("json"),
+                .hide(true),
         )
         .get_matches();
 
+    if matches.is_present("inputFormat") {
+        eprintln!("--inputFormat (-f) is a no-op and will be removed in a future version. All input is treated as YAML");
+        eprintln!("as it is a strict super-set of JSON (all valid JSON is valid YAML).");
```

Comment:
Note: YAML was added last month, and isn't actually published in a new release. Not sure we need the deprecation notice, but it is a good exercise.

Reply:
I think this works for both json and yaml if I'm not mistaken